gpii/node_modules/matchMakerFramework/src/MatchMakerUtilities.js

```diff
@@ -175,6 +188,135 @@ var fluid = fluid || require("infusion"),
         return togo;
     };
 
+    // true if one of the values matches across the two arrays (ignoring the indexing)
+    gpii.matchMakerFramework.utils.containsSameValue = function (arr1, arr2) {
+        var obj1 = {};
+        var i;
+        // convert arr1 to object for fast and easy comparison
+        for (i in arr1) {
+            obj1[arr1[i]] = true;
+        }
+        // compare:
+        for (i in arr2) {
+            if (arr2[i] in obj1) {
+                return true;
+            }
+        }
+        return false;
+    };
+
+    /**
+     */
+    gpii.matchMakerFramework.utils.disposeFromPriority = function (solrecs, data) {
+        // sort into prioritized map: 1024: [sola, solb], 200 [solc]
+        var prioritiesMap = {};
+        fluid.each(solrecs, function (sol) {
+            if (sol.priority) {
+                fluid.set(prioritiesMap, [sol.priority, sol.index], true);
+            }
+        });
+        // sort priority entries desc
+        var priorities = Object.keys(prioritiesMap).sort(function (a, b) { return b - a; });
+        // for each priority bucket:
+        for (var i = 0; i < priorities.length; i++) {
+            var priority = priorities[i];
+            var sols = prioritiesMap[priority];
+
+            // for each solution as sol
+            for (var solutionId in sols) {
+                var tied = [];
+                var solutionType = data.solutionTypes[solutionId];
+                // if a disposition is already decided for this solution, or if priority has been
+                // removed (in case of ties), ignore it.
+                if (solrecs[solutionId].disposition || !solrecs[solutionId].priority) {
+                    continue;
+                }
+                // check if there are any similar typed solution with same priority:
+                // for each remaining solution as rsol
+                // delete sols[solutionId];
```

Comment:
I forget exactly why I put them there in the first place, actually.... But i remember screwing something up when adding support for tied solutions, and I think this might be a leftover from that.